ofl/commissioner/METADATA.pb

```diff
@@ -0,0 +1,36 @@
+name: "Commissioner"
+designer: "Kostas Bartsokas"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2020-07-20"
+fonts {
+  name: "Commissioner"
+  style: "normal"
+  weight: 400
+  filename: "Commissioner[FLAR,VOLM,slnt,wght].ttf"
+  post_script_name: "Commissioner-Thin"
+  full_name: "Commissioner Thin"
+  copyright: "Copyright 2019 The Commissioner Project Authors (https://github.com/kosbarts/Commissioner)"
+}
+subsets: "cyrillic"
+subsets: "cyrillic-ext"
+subsets: "greek"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "vietnamese"
+axes {
```

Comment:
@rsheeter Since these axes are unregistered, shall we remove them and only keep the wght axis?

Reply:
I see Nate recently PR'd the axisregistry. Should we make sure that our METADATA.pb files don't include axes which are not part of this doc?